### ISSUE TITLE
is there a specific reason for not supporting mapbox-gl > 0.41?

### ISSUE THREAD

**Issue by bartvde** [mapbox/mapbox-gl-draw#729]
Currently the peerDependency for mapbox-gl is pretty strict. Is there a specific reason that 0.42 and later are not supported? I couldn't find any information on this. Or is it simply because this package was released when 0.41.0 was current?

```
  "peerDependencies": {
    "mapbox-gl": ">=0.27.0 <=0.41.0"
  },
```

TIA for any feedback.

**Comment by Methodician** [comment]
I'm having all sorts of conflicts because of this. Latest versions of other packages require latest version of mapbox-gl so I'm having to roll back several things just to support drawing tools!

**Comment by mcwhittemore** [comment]
Yea. Sorry. This is a pain for both users and maintainers and fixing it would be great. It is set up this way currently because minor changes in mapbox-gl can be breaking and have broken versions of mapbox-gl-draw in the past. 

That said, it might be worth hearing from someone working on mapbox-gl about how stable the features used by draw are and if we can remove this.

**Comment by jasonbellis** [comment]
Like @Methodician said, 0.43.0 appears safe for now. From personal experience version 0.42.0 is also good, 0.42.1 and 0.42.2 had some breaking changes regarding selecting draw features. We're pushing mapbox-gl and mapbox-gl-draw to the nth degree in our app and the features in 0.43.0 were worth the risk to move up to. We've been good so far.

**Comment by lgzajac** [comment]
any news about updating dependencies? ;)
most of the pull requests are proposed by greenkeeper bot..

**Comment by mcwhittemore** [comment]
I've just merged the greenkeeper PRs.

Per mapbox-gl-js dependency, there are some breaking changes between 0.41 and 0.44 that would be great to review so we can update that dependency.

**Comment by mcwhittemore** [comment]
This has been resolved and we now support <= 0.45

**Comment by bartvde** [comment]
thank you @mcwhittemore
